source/common/network/win32_redirect_records_option_impl.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
```

Comment:
Can/should this class/files live in the original_dst filter?

Reply:
Wouldnt that create an awkward dependency graph?

`\\source\common\network:socket_option_factory_lib` depends on `\\source\extension\filters\listener\filters\original_dst:win32_redirect_records_option_lib`